layouts/security/list.html

```diff
@@ -2,8 +2,8 @@
 
 <div class="body">
     <main role="main">
+        {{ partial "seo/structured/breadcrumbs.html" . }}
         <article class="static doc">
-
```

Comment:
Usually, don't add or remove empty lines unless it is really needed. It makes the review process simpler if we only see what is really changing something.